lib/index.js

```diff
@@ -17,7 +17,7 @@ exports.plugin = {
     multiple: true,
     pkg: require('../package.json'),
     requirements: {
-        hapi: '>=17.7.0'
+        hapi: '>=18.0.0'
```

Comment:
Oh boy!  This actually doesn't need updating, does it?  This change would drop support of v17.

Reply:
Ohp! Goodness, this was a mistake, definitely don't want to drop support for v17